<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateOrderForm</class>
 <widget class="QWidget" name="CreateOrderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreateOrderForm</string>
  </property>
  <widget class="QGroupBox" name="descriptionGroupBox">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>100</y>
     <width>491</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Описание позиции заказа</string>
   </property>
   <widget class="QTextEdit" name="descriptionTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>471</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="remarkGroupBox">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>530</y>
     <width>841</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Примечание к заказу</string>
   </property>
   <widget class="QTextEdit" name="remarkTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>821</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>210</y>
     <width>248</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="addPositionBtn">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>Добавить позицию</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delPositionBtn">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>Удалить позицию</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="quantityGroupBox">
   <property name="geometry">
    <rect>
     <x>517</x>
     <y>100</y>
     <width>120</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Количество</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="quantityLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="positionsGroupBox">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>270</y>
     <width>841</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Позиции заказа</string>
   </property>
   <widget class="QTableWidget" name="positionTableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>821</width>
      <height>201</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Описание</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Количество</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Выдача</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>660</y>
     <width>841</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="addOrderBtn">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Внести заказ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Сохранить изменения</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelBtn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="issueGroupBox">
   <property name="geometry">
    <rect>
     <x>657</x>
     <y>100</y>
     <width>120</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Выдача</string>
   </property>
   <widget class="QComboBox" name="issueComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1184</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item row="0" column="5">
     <widget class="QLabel" name="designerLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Дизайнер</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="dateLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Дата</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="managerLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Менеджер</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="amountLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сумма</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="paymentLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Оплата</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="clientLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Клиент</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="amountLineEdit">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="paymentComboBox">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QComboBox" name="managersComboBox"/>
    </item>
    <item row="1" column="5">
     <widget class="QComboBox" name="designersComboBox"/>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="QDateEdit" name="dateEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openCalendarBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QComboBox" name="clientsComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="maxVisibleItems">
         <number>15</number>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>openCalendarBtn</sender>
   <signal>clicked()</signal>
   <receiver>CreateOrderForm</receiver>
   <slot>slot1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addPositionBtn</sender>
   <signal>clicked()</signal>
   <receiver>CreateOrderForm</receiver>
   <slot>slot3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delPositionBtn</sender>
   <signal>clicked()</signal>
   <receiver>CreateOrderForm</receiver>
   <slot>slot4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addOrderBtn</sender>
   <signal>clicked()</signal>
   <receiver>CreateOrderForm</receiver>
   <slot>slot2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>688</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>715</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveBtn</sender>
   <signal>clicked()</signal>
   <receiver>CreateOrderForm</receiver>
   <slot>slot5()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>680</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>715</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>CreateOrderForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>808</x>
     <y>682</y>
    </hint>
    <hint type="destinationlabel">
     <x>695</x>
     <y>713</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clientsComboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>CreateOrderForm</receiver>
   <slot>slot7()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot1()</slot>
  <slot>slot2()</slot>
  <slot>slot3()</slot>
  <slot>slot4()</slot>
  <slot>slot5()</slot>
  <slot>slot6()</slot>
  <slot>slot7()</slot>
 </slots>
</ui>
